sycl/include/CL/sycl/detail/pi.h

```diff
@@ -246,7 +247,10 @@ static const uint8_t PI_DEVICE_BINARY_OFFLOAD_KIND_SYCL = 4;
 ///
 #define PI_DEVICE_BINARY_TARGET_UNKNOWN "<unknown>"
 #define PI_DEVICE_BINARY_TARGET_SPIRV32 "spir"
-#define PI_DEVICE_BINARY_TARGET_SPIRV64 "spir64";
+#define PI_DEVICE_BINARY_TARGET_SPIRV64 "spir64"
+#define PI_DEVICE_BINARY_TARGET_SPIRV64_X86_64 "spir64_x86_64"
+#define PI_DEVICE_BINARY_TARGET_SPIRV64_GEN "spir64_gen"
+#define PI_DEVICE_BINARY_TARGET_SPIRV64_FPGA "spir64_fpga"
```

Comment:
These are not directly used by PI API, so we need to explain in the piextDeviceSelectBinary description how these are used. Also let's document what each of these represent in particular.

Reply:
>> Also let's document what each of these represent in particular.
I'm a little bit confused with this one. For example, would such comments suffice:
```
// SPIR: standard portable IR for OpenCL 64-bit version
#define PI_DEVICE_BINARY_TARGET_SPIRV64 "spir64"
```

Reply:
yes, it would be OK
more importantly describe where these are used.

Reply:
Please take a look at the updated comment sections - I'm not entirely convinced by these myself.

Reply:
These comments look good, but please update the description of how/where these are used. Today it is not clear from this:
~~~
  /// null-terminated string representation of the device's target architecture
  const char *DeviceTargetSpec;	  const char *DeviceTargetSpec;
~~~

Reply:
@smaslov-intel Done.